### PR TITLE
MINOR: Adding deprecated KTable methods to docs from KIP-114

### PR DESCRIPTION
**Pull request by bbejeck**

### CODE REVIEW

**Comment by bbejeck** [comment]
@dguy @mjsax @guozhangwang @miguno for review

File: docs/streams.html [hunk 1]
```diff
@@ -836,6 +836,32 @@
...
+            <li><code>void print(final Serde&lt;K&gt; keySerde, final Serde&lt;V&gt; valSerde)</code></li>
+            <li><code>void print(final Serde&lt;K&gt; keySerde, final Serde&lt;V&gt; valSerde, final String streamName)</code></li>
+            <li><code>void writeAsText(final String filePath)</code></li>
+            <li><code>void writeAsText(final String filePath, final Serde&lt;K&gt; keySerde, final Serde&lt;V&gt; valSerde)</code></li>
+            <li><code>void writeAsText(final String filePath, final String streamName)</code></li>
+            <li><code>void writeAsText(final String filePath, final String streamName, final Serde&lt;K&gt; keySerde, final Serde&lt;V&gt; valSerde)</code></li>
+        </ul>
+
+        <p>
+             These methods have been deprecated in favor of using the Interactive Queries API.
```
Comment (guozhangwang) [inline docs/streams.html]: We should not say it is `the equivalent functionality`: the deprecated function is for printing the changelog stream of the KTable, and can still be executed by table.toStream.XX.

We can re-word the IQ part as If you want to query the current content of the state store backing the KTables, use the following approach ...

**Comment by bbejeck** [comment]
@guozhangwang updates per comments, rebased PR

File: docs/streams.html [hunk 2]
```diff
@@ -850,7 +850,39 @@
...
+        Deprecated methods in <code>KTable</code>
+        </p>
+        <ul>
+            <li><code>void foreach(final ForeachAction&lt;? super K, ? super V&gt; action)</code></li>
+            <li><code>void print()</code></li>
+            <li><code>void print(final String streamName)</code></li>
+            <li><code>void print(final Serde&lt;K&gt; keySerde, final Serde&lt;V&gt; valSerde)</code></li>
+            <li><code>void print(final Serde&lt;K&gt; keySerde, final Serde&lt;V&gt; valSerde, final String streamName)</code></li>
+            <li><code>void writeAsText(final String filePath)</code></li>
+            <li><code>void writeAsText(final String filePath, final Serde&lt;K&gt; keySerde, final Serde&lt;V&gt; valSerde)</code></li>
```
Comment (guozhangwang) [inline docs/streams.html:863]: Switch line 863 with 864

File: docs/streams.html [hunk 3]
```diff
@@ -850,7 +850,39 @@
             <li> added overloads for <code>#globalKTable()</code> that allow to define a <code>TimestampExtractor</code> per global table </li>
         </ul>
 
-        <h3><a id="streams_api_changes_01021" href="#streams_api_changes_01021">Stream API changes in 0.10.2.1</a></h3>
+        <p>
```
Comment (guozhangwang) [inline docs/streams.html:853]: Need to add `</li>` on 836 and above

File: docs/streams.html [hunk 4]
```diff
@@ -850,7 +850,39 @@
...
+           If you want to query the current content of the state store backing the KTable, use the following approach:
+             <ul>
+               <li> Make a call to <code>KafkaStreams.store(final String storeName, final QueryableStoreType&lt;T&gt; queryableStoreType)</code> followed by a call to <code>ReadOnlyKeyValueStore.all()</code> to iterate over the keys of a <code>KTable</code>.
+               </li>
+            </ul>
+        </p>
+        <p>
+           If you want to view the changelog stream of the  <code>KTable</code> then you could do something along the lines of the following:
+         <ul>
+           <li> Call <code>KTable.toStream()</code> then call <code>KStream#print()</code>.</li>
```
Comment (guozhangwang) [inline docs/streams.html:881]: Merge it to `KTable.toStream().print()`.

File: docs/streams.html [hunk 5]
```diff
@@ -850,7 +850,39 @@
...
+        </ul>
+        </p>
+
+        <h3><a id="streams_api_changes_01021" href="#streams_api_changes_01021">Notable changes in 0.10.2.1</a></h3>
```
Comment (guozhangwang) [inline docs/streams.html:885]: Reword the line 891 a bit.

File: docs/streams.html [hunk 6]
```diff
@@ -850,7 +850,39 @@
...
+               <li> Make a call to <code>KafkaStreams.store(final String storeName, final QueryableStoreType&lt;T&gt; queryableStoreType)</code> followed by a call to <code>ReadOnlyKeyValueStore.all()</code> to iterate over the keys of a <code>KTable</code>.
+               </li>
+            </ul>
+        </p>
+        <p>
+           If you want to view the changelog stream of the  <code>KTable</code> then you could do something along the lines of the following:
+         <ul>
+           <li> Call <code>KTable.toStream()</code> then call <code>KStream#print()</code>.</li>
+        </ul>
+        </p>
```
Comment (guozhangwang) [inline docs/streams.html:883]: Move `</p>` here at 880 before `<ul>`.

**Review by guozhangwang — COMMENTED** [review]
LGTM. Merged to trunk and 0.11.0 with some minor tweaks squashed into the commit (details listed).

**Pull request closed without merging**